### PR TITLE
Halves volume of bluespace beakers

### PR DESCRIPTION
**Pull request by 5dplanner**
## About The Pull Request
Halves volume of bluespace beakers. They are now a useful but not massive upgrade over large beakers.

## Why It's Good For The Game
Bluespace beakers have always been overpowered but overlooked and are the root cause of most problems with chemistry explosives.

## Changelog
:cl:
balance: Halved volume of bluespace beakers
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/glass.dm [hunk 1]
```diff
@@ -208,9 +208,9 @@
 		300 units."
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/reagent_containers/glass.dm]: 🤔

**Comment by Rowelll** [comment]
you
are kidding
right?

**Comment by 5dplanner** [comment]
> 
> 
> you
> are kidding
> right?

I never 'kid'

**Comment by Rowelll** [comment]
> > you
> > are kidding
> > right?
> 
> I never 'kid'

then start

**Comment by yoyobatty** [comment]
Funny how spray bottles and the medical design beakers will still surpass the carrying capacity.

**Comment by 5dplanner** [comment]
> 
> 
> Funny how spray bottles and the medical design beakers will still surpass the carrying capacity.

So you wish for a comprehensive rebalance of reagent containers based around the new maximum capacity of 150? That can be arranged...

**Comment by dr-bee** [comment]
This would be great for reducing the power of chem grenades but a pain for chemical mass production, which is the primary use of those beakers.

Restricting grenades to large beakers at maximum would be a good alternative change, or better yet, adding a non-portable method of mixing large amounts of chemicals like a barrel or vat.

**Comment by yoyobatty** [comment]
Well isn't it just ironic that you balance one with the other? Go ahead and do this though because we're just going to revert it when it comes to hippie, I just want to make sure that you guys are being consistent with your own stuff.

**Comment by BuffEngineering** [comment]
This makes bluespace beakers only 30u better than x-large beakers (which only cost plastic and glass to make), and actually makes them worse than metamaterial beakers which are gold tier. If nerfing bluespace beaker capacity is what needs to happen they should either be 200u to fit in with the current tiers of beakers or all beakers should be adjusted.

**Comment by QualityVan** [comment]
> This would be great for reducing the power of chem grenades but a pain for chemical mass production, which is the primary use of those beakers.
Isn't that explicitly why fermenting barrels were added?

**Comment by ATH1909** [comment]
> This would be great for reducing the power of chem grenades but a pain for chemical mass production, which is the primary use of those beakers.
> 
> Restricting grenades to large beakers at maximum would be a good alternative change, or better yet, adding a non-portable method of mixing large amounts of chemicals like a barrel or vat.

This'd be a much better change, IMO.

**Comment by 81Denton** [comment]
This makes bluespace beakers worse than meta beakers.
Better solution would be to add a bluespace crystal cost to them and/or restricting them to large grenade casings instead of regular roundstart grenade casings.

Otherwise it's a bit silly that a beaker from the 2500pts advanced biotech node is worse than one from the 5000pts applied bluespace node.

**Comment by wesoda25** [comment]
Make it so that they don’t fit into any type of grenade.

**Comment by fludd12** [comment]
@qualityvan Fermenting barrels were added to make alcohol, I only added reagent storage because making water barrels or powder kegs sounded cool.

**Comment by DapperMola** [comment]
-5dplanner
-nerfing something that doesnt need to be nerfed

:thinking:

**Comment by Timonkeyn** [comment]
2 bluespace beakers with potassium and water in an IED Lance barely rip a 3x3 hole, why?

I'm also sure that bluespace beakers are supposed to be mid-end game, do not fucking merge. 5d literally joined 10 days ago.

**Comment by DapperMola** [comment]
>5d literally joined 10 days ago.

Literally just a gag account to make fun of 4d. A hearty kek.

**Comment by Timonkeyn** [comment]
If we pretend we don't know who 4d is this is still a bad PR

**Comment by CthulhuOnIce** [comment]
Bluespace can be used for teleportation, but holding more than 150 units? No way

**Comment by DapperMola** [comment]
A theoretically infinitely large backpack: Yeah okay sure
B ig Cu  p: Banned.

**Comment by HideAndSeekLOGIC** [comment]
Considering that chem dispensers aren't infinite, bluespace beakers are incredibly pricey and even a 600u explosion isn't actually that big this is a shitty nerf no-one asked for.

-1.

**Comment by yoyobatty** [comment]
I mean too be fair, upgrading a Chem dispenser with a t4 part replacer with bluespace cells will make them last quite a long time

**Comment by Timonkeyn** [comment]
Even 4d dislikes this

**Comment by imsxz** [comment]
i love 5dplanner

big cup is the root of all evil kill it NOW only small cup

**Comment by Dorsisdwarf** [comment]
Fuck off you bastard

**Comment by OverlyGeneric** [comment]
lazy form of nerfing. your younger brother is better at this, 5dplanner. 👎

**Comment by necromanceranne** [comment]
> i love 5dplanner
> 
> big cup is the root of all evil kill it NOW only small cup

Everyone knows big cups are best because they can contain more milk.

**Comment by optimumtact** [comment]
I'm gonna merge it

**Comment by necromanceranne** [comment]
Everyone forgets about metabeakers....

**Comment by optimumtact** [comment]
I already removed them in an earlier PR, it hasn't hit main yet

**Comment by JMoldy** [comment]
Why not just make normal grenades only able to accept reagent containers of volume <151?

**Comment by epicmemename** [comment]
✔️based
✔️redpilled
*ahem*
FUCK CHEMFAGS, FUCK SCIENCE, AND FUCK REDDIT

**Comment by carshalash** [comment]
I guess everyone will just abuse the monkey bombs now

**Comment by Timonkeyn** [comment]
Are you fucking kidding me? Literally 24 thumbs down.

**Comment by Timonkeyn** [comment]
Okay so I heard that oranges stealth reverted it because it was a joke.

**Comment by 81Denton** [comment]
rused

**Comment by Timonkeyn** [comment]
It's funny how the bots still mirror it

**Comment by ExcessiveUseOfCobblestone** [comment]
>

**Comment by DapperMola** [comment]
i'm having a fucking stroke, this is such a roller coaster of emotions

**Comment by Bawhoppen** [comment]
oranges just likes to show off the unlimited political capital they wield over the codebase

**Comment by ExcessiveUseOfCobblestone** [comment]
> everyone complains about le epic 300u grenades whenever something remotely occurs with chemicals
>> it gets nerfed
>>> wtf hitler!!

**Pull request merged**